src/generators/icons/index.ts

```diff
@@ -36,7 +43,14 @@ const items = [
     require("./items/document"),
     require("./items/documentBinary"),
     require("./items/downloadedDocument"),
-    require("./items/documentBinaryDownloaded")
+    require("./items/documentBinaryDownloaded"),
+    require("./items/pattern"),
+    require("./items/session"),
+    require("./items/sessionOpen"),
+    require("./items/sessionFavourite"),
+    require("./items/sessionFavouriteOpen"),
+    require("./items/folder"),
+    require("./items/folderOpen")
```

Comment:
It'd be nice if we could read all icons in a directory and load them all instead of having to modify this file every time a new icon is added.

No changes requested at this time though! :smile: